storage-units/src/indexfile.rs

```diff
@@ -198,6 +211,14 @@ impl Index {
             write_offset(&mut buf, ofs);
             tmpfile.write_all(&buf[..])?;
         }
+
+        // Write ordered offsets to seek blocks by height
+        for ofs in self.offsets.iter() {
+            let mut buf = [0u8; OFF_SIZE];
+            write_offset(&mut buf, *ofs);
+            tmpfile.write_all(&buf[..])?;
+        }
```

Comment:
This logic will be moved to another file (https://github.com/input-output-hk/rust-cardano/blob/master/storage/src/config.rs#L72), we will not extend the index field for now